### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by chalkera**
Created some documentation for reducers and actions and created a footer to set up a pagination

### CODE REVIEW

File: client/src/SearchBar.jsx [hunk 1]
```diff
@@ -7,25 +7,42 @@ const SearchBar = props => (
       id="location"
       type="text"
       placeholder="location"
+<<<<<<< HEAD
```
Comment (mikehern) [inline client/src/SearchBar.jsx]: Seeing git diff text injected.  Anyone else see it?  Needs cleanup?

**Review by mikehern — CHANGES_REQUESTED** [review]

File: client/src/SearchBar.jsx (hunk 1 above)
Comment (chalkera) [reply client/src/SearchBar.jsx]: fixed it
look again

**Review by kfang1220 — COMMENTED** [review]
Looks fine to me

File: client/src/SearchBar.jsx (hunk 1 above)
Comment (kfang1220) [reply client/src/SearchBar.jsx]: He changed them

**Pull request merged**